docs/pages/index.mdx

```diff
@@ -6,102 +6,72 @@ layout: tocless-doc
 videoBanner: fSgCnQrNu78
 ---
 
-Teleport is a certificate authority and access plane for your infrastructure.
-With Teleport you can:
+Teleport is Identity-Native Infrastructure Access for engineers and machines. With Teleport you can:
```

Comment:
Would it make more sense to call Teleport an Identity-Native Infrastructure Access _platform_? I think the statement of what Teleport is can be more concrete.